### PR TITLE
Javascript/javascript2/week1

### PR DESCRIPTION
**Pull request by GreeshmaRahool**
Added Javascript 2 week 1 homework.

### CODE REVIEW

**Comment by wwang107** [comment]
Learn the following git commands, `rebase`, `reset --soft HEAD^1` and `push -f`, to eliminate mistake pushed commit

File: javascript/javascript2/week1/ShortestWord.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    let wordToCompare = danishWords[0].length;
+    for(let i = 0; i < danishWords.length; i ++)
+    {
+        if(wordToCompare >= danishWords[i].length)
+        {
+            wordToCompare = danishWords[i].length;
+            shortestWord =   danishWords[i];
+        }
+    }
+    return shortestWord;
```
Comment (wwang107) [inline javascript/javascript2/week1/ShortestWord.js:15]: `danishWords.forEach(word=>{...})`

File: javascript/javascript2/week1/SpiritAnimalGenerator/spiritAnimalGenerator.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+const arrSpiritAnimals = ['The beautiful swan','The graceful peacock','The dark moon panther','The confident jaguar','The playful cayote','The creative eagle','The joyful hummingbird','The energetic horse','The innocent deer','The patient spider'];
+
+//Create an input element, a button and a tag to display the spirit animal into.
+const textBox = document.createElement('input');
+textBox.setAttribute('Id','txtName');
+textBox.setAttribute('placeholder','Enter your name');
```
Comment (wwang107) [inline javascript/javascript2/week1/SpiritAnimalGenerator/spiritAnimalGenerator.js:6]: Nice touch

File: javascript/javascript2/week1/SpiritAnimalGenerator/spiritAnimalGenerator.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+const arrSpiritAnimals = ['The beautiful swan','The graceful peacock','The dark moon panther','The confident jaguar','The playful cayote','The creative eagle','The joyful hummingbird','The energetic horse','The innocent deer','The patient spider'];
+
+//Create an input element, a button and a tag to display the spirit animal into.
+const textBox = document.createElement('input');
```
Comment (wwang107) [inline javascript/javascript2/week1/SpiritAnimalGenerator/spiritAnimalGenerator.js:4]: `inputBox` or simply `input` would be more descriptive

File: javascript/javascript2/week1/SpiritAnimalGenerator/spiritAnimalGenerator.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+{
+    const inputName = textBox.value;
+
+    if(inputName)
+    {
+        para.innerHTML = `${inputName} - ${arrSpiritAnimals[parseInt(Math.random() * 10)]}`;
+    }
+    else
+    {
+        para.innerHTML = `Please provide a name!`;
```
Comment (wwang107) [inline javascript/javascript2/week1/SpiritAnimalGenerator/spiritAnimalGenerator.js:56]: `para.innerHTML = inputName ? "${inputName} - ${arrSpiritAnimals[parseInt(Math.random() * 10)]}" : "Please provide a name!"`

File: javascript/javascript2/week1/SpiritAnimalGenerator/index.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+   
+        <select id="selectEvent" onchange="onSelectChange()">
+            <option value="Select">Select</option>
+            <option value = "Click" id="clickMe">Click Button</option>
+            <option value = "Hover" id="hoverme">Hover on Button</option>
+        </select>
+
+      </form> 
+    </div>
+    <script src="spiritAnimalGenerator.js"></script>
```
Comment (wwang107) [inline javascript/javascript2/week1/SpiritAnimalGenerator/index.html:28]: Just curious. Why didn't you create some template html, such as input and dropdown here?

**Review by wwang107 — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week1/SpiritAnimalGenerator/index.html (hunk 5 above)
Comment (GreeshmaRahool) [reply javascript/javascript2/week1/SpiritAnimalGenerator/index.html:28]: I didn't understand what you meant here.

Comment (GreeshmaRahool) [reply javascript/javascript2/week1/SpiritAnimalGenerator/index.html:28]: Thanks for the comments. Can you help me with an example?I am not sure if I'm getting this right.

**Pull request merged**